resources/js/components/ModalManageTimetable.vue

```diff
@@ -0,0 +1,242 @@
+<template>
+    
+
+
+    <div class="fixed z-10 inset-0 overflow-y-auto">
+        <div class="flex items-center justify-center min-h-screen pt-4 px-4 text-center sm:block sm:p-0">
+            <div class="fixed inset-0 bg-black opacity-50"></div>
+            <div class="bg-white p-8 rounded-lg z-50 shadow-md max-w-sm mx-auto relative">
+                <!-- Pulsante di chiusura del modal -->
+                <button @click="this.$emit('closeModal')" class="absolute top-0 right-0 m-4 text-gray-600 hover:text-gray-800 focus:outline-none">
+                    <svg class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                        <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M6 18L18 6M6 6l12 12"></path>
+                    </svg>
+                </button>
+                <!-- Contenuto del modal -->
+                <h2 class="text-lg font-semibold mb-4">Modifica l'orario</h2>
```

Comment:
Non dovrebbe chiamarsi "Modifica del docente e materia" in quanto da quello che vedo non modifichi l'ora

Reply:
modificato, forse è più chiaro